src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    //  robot parameters
+
+    rN: name, // rN = robotName
+    rW: wheels, // rW = robotWheels
+    rV: version, // rV = robotVersion
```

Comment:
```suggestion
    name,
    wheels,
    version,
```
зачем делать какие-то rN/rW?